docs-source/content/userguide/managing-fmw-domains/soa-suite/WebLogic-Monitoring-Exporter-Setup.md

```diff
@@ -0,0 +1,298 @@
+# WebLogic Monitoring Exporter Setup to Monitor the SOA Domain Instance
```

Comment:
WebLogic Monitoring Exporter Setup to Monitor the SOA Domain Instance -> Use the WebLogic Monitoring Exporter to monitor a SOA domain